1.4.3

asilverthorn opened the pull request:
Small bug-fix build. Includes:

- Me/TheShwantz27: https://trello.com/c/DFto69Cj/79-scan-flag-does-not-get-logged
- TheShwantz27: https://trello.com/c/8PSJxWpv/80-coliseum-opponents-rewards-wrong-in-track-menu
- Me: https://github.com/ff6wc/WorldsCollide/pull/125 (PT multi-award)
- Me: No Free Character/Espers (aka No Free Progression) 100% is now completable with the removal of Mt. Zozo from the list
- rblakedavis: https://github.com/ff6wc/WorldsCollide/pull/122 (new output option)
- HoxNorf: https://github.com/ff6wc/WorldsCollide/pull/123 (new sprites)
- dbldown11: https://github.com/ff6wc/WorldsCollide/pull/131 (Dried Meat changes)
- now works with Python 3.12+

HoxNorf committed:
Small Aesthetic Additions

Palettes Added (10):
- Alice-HoxNorf-Touhou
- Alphys-LoneRedMage-Undertale
- Amy-HoxNorf-Sonic
- Frisk-LoneRedMage-Undertale
- Gryz-HoxNorf-PS4
- Raja-HoxNorf-PS4
- Reimu (Blue)-HoxNorf-Touhou
- Rune-HoxNorf-PS4
- Toriel-LoneRedMage-Undertale
- Yuyuko-HoxNorf-Touhou

Portraits Added (20):
- Alice-HoxNorf-Touhou
- Alma-Unknown-FFT
- Alphys-LoneRedMage-Undertale
- Amy-HoxNorf-Sonic
- Cloud-JamesWhite89-FF7
- Fighter-CtrlxZ-FF1
- Frisk-LoneRedMage-Undertale
- Gryz-HoxNorf-PS4
- Jessie-JamesWhite89-FF7
- Lenna (White Mage)-JamesWhite89-FF5
- Lenna-JamesWhite89-FF5
- Orlandeau-Unknown-FFT
- Porom (Devout)-HoxNorf-FF4
- Raja-HoxNorf-PS4
- Ramza-Unknown-FFT
- Reimu (Blue)-HoxNorf-Touhou
- Rune-HoxNorf-PS4
- Sarisa-JamesWhite89-FF5
- Toriel-LoneRedMage-Undertale
- Yuyuko-HoxNorf-Touhou

Sprites Added (9):
- Alice-HoxNorf-Touhou
- Alphys-LoneRedMage-Undertale
- Amy-HoxNorf-Sonic
- Frisk-LoneRedMage-Undertale
- Gryz-HoxNorf-PS4
- Raja-HoxNorf-PS4
- Rune-HoxNorf-PS4
- Toriel-LoneRedMage-Undertale
- Yuyuko-HoxNorf-Touhou

* Renamed Pirahna Plant-JamesWhite89-Mario to Piranha Plant-JamesWhite89-Mario

HoxNorf committed:
Sprite Additions and "Restorations"

- New Sprites/Palettes (11):
Arthur-JamesWhite89-GnG
Cait Sith-HoxNorf-FF7
Donkey Kong-Badass-Mario
Gilius-JamesWhite89-GoldenAxe
Kain (Holy Dragoon)-CtrlxZ-FF4TAY
Lugae-Astaroth-FF4
Nitori-HoxNorf-Touhou
Sanae-HoxNorf-Touhou
Sherlotta-HoxNorf_ScarabEnigma-FFCC
Wren-HoxNorf-PS4
X-Badass-Megaman

- New Portraits (30):
Arthur-JamesWhite89-GnG
Banon-Laurel_Gens-FF6PR
Cait Sith-HoxNorf-FF7
Celes-Laurel_Gens-FF6PR
Cyan-Laurel_Gens-FF6PR
Donkey Kong-Badass-Mario
Edgar-Laurel_Gens-FF6PR
Gau-Laurel_Gens-FF6PR
General Leo-Laurel_Gens-FF6PR
Gilius-JamesWhite89-GoldenAxe
Ghost-Laurel_Gens-FF6PR
Gogo-Laurel_Gens-FF6PR
Imp-Laurel_Gens-FF6PR
Kain (Holy Dragoon)-Unknown-FF4TAY
Locke-Laurel_Gens-FF6PR
Lugae-HoxNorf-FF4
Mog-Laurel_Gens-FF6PR
Nitori-HoxNorf-Touhou
Relm-Laurel_Gens-FF6PR
Sabin-Laurel_Gens-FF6PR
Sanae-HoxNorf-Touhou
Setzer-Laurel_Gens-FF6PR
Shadow-Laurel_Gens-FF6PR
Sherlotta-HoxNorf-FFCC
Strago-Laurel_Gens-FF6PR
Terra-Laurel_Gens-FF6PR
Umaro-Laurel_Gens-FF6PR
WedgeVicks-Laurel_Gens-FF6PR
Wren-HoxNorf-PS4
X-Unknown-Megaman

- "Restored" Sprites/Palettes (32):
Antlion-Astaroth-FF4
Atma-Astaroth-FF6
Boy-Zozma-FF6
Cagnazzo-Astaroth-FF4
Celes (Amano)-Astaroth-FF6
Celes (Opera)-Astaroth-FF6
Clyde-PocoLoco-FF6
Cultist-PocoLoco-FF6
Dancer-PocoLoco-FF6
Dark Elf-Astaroth-FF4
Draco-PocoLoco-FF6
Elena-Astaroth-FF7
Figaro Guard-PocoLoco-FF6
Interceptor-JamesWhite89-FF6
Katarin-Zozma-FF6
Link-FEOK-LegendOfZelda
LoneWolf-PocoLoco-FF6
Lucca-FEOK-CT
Lufia-JamesWhite89-Lufia
Maduin-PocoLoco-FF6
Mini-JamesWhite89-FF
NarsheGuard-PocoLoco-FF6
Peach-Halkel-SMRPG
Rubicante-Astaroth-FF4
Scholar-PocoLoco-FF6
Siegfried-PocoLoco-FF6
Squall (Uniform)-SApprentice-FF8
Squall-PocoLoco-FF8
Tifa-Astaroth-FF7
Ultros-PocoLoco-FF6
Vargas-PocoLoco-FF6
Vincent-FEOK-FF7

- "Restored" Portraits (3):
Golbez_TAY-HoxNorf-FF4
Link-JamesWhite89-LegendOfZelda
Vincent-Xeblon-FF7

* Removed Duplicate Entries in Python Files
* Gave proper credit to the Lufia and Vincent portraits

HoxNorf committed:
New Aesthetic Additions

New Sprites/Palettes (11):
- Aria-HoxNorf-FF3
- Arle-HoxNorf-Madou
- Marle-CtrlxZ-CT
- Nakoruru-HoxNorf-SamSho
- Noire-HoxNorf-Neptunia
- Purple Sister-HoxNorf-Neptunia
- Rash-JamesWhite89-Battletoads
- Refia-HoxNorf-FF3
- Relm-HoxNorf-FF6PR
- Simon-JamesWhite89-Castlevania
- Tina-CtrlxZ-FF6

New Portraits (14):
- Aria-JamesWhite89-FF3
- Arle-HoxNorf-Madou
- Magus-asilverthorn-CT
- Marle-CtrlxZ-CT
- Nakoruru-HoxNorf-SamSho
- Noire-JamesWhite89-Neptunia
- Purple Sister-HoxNorf-Neptunia
- Rash-JamesWhite89-Battletoads
- Refia-JamesWhite89-FF3
- Santa_Olaf-asilverthorn-AW
- Simon-JamesWhite89-Castlevania
- Tina-CtrlxZ-FF6
- Vivi-asilverthorn-FF9
- Zidane-asilverthorn-FF9

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
New Portraits by asilverthorn

New Portraits (2):
- Cloud-asilverthorn-Dissidia
- Sephiroth-asilverthorn-Dissidia

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
Restorations and Additions

New Sprites/Palettes (2):
- Luneth-HoxNorf-FF3
- Malin-HoxNorf-KOF

New Portraits (8):
- Barret-HoxNorf-FF7
- Garnet (Long Hair)-HoxNorf-FF9
- Garnet (Short Hair)-HoxNorf-FF9
- Lightning-asilverthorn-Dissidia
- Luneth-JamesWhite89-FF3
- Malin-HoxNorf-KOF
- Steiner-HoxNorf-FF9
- Yuffie-CtrlxZ-KH

"Restored" Portraits (3):
- Figaro Guard-JamesWhite89-FF6
- Simon-JamesWhite89-Castlevania
- Vincent-Xeblon-FF7

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
New Portraits by Jones and Gens

New Portraits (14):
- Bartz-metalliguy_Gens-FF5
- Celes-Jones-FF6
- Chocobo-metalliguy_Gens-FF5
- Edgar-Jones-FF6
- Faris-metalliguy_Gens-FF5
- Galuf-metalliguy_Gens-FF5
- Gau-Jones-FF6
- Krile-metalliguy_Gens-FF5
- Lenna-metalliguy_Gens-FF5
- Locke-Jones-FF6
- Relm-Jones-FF6
- Sabin-Jones-FF6
- Terra-Jones-FF6
- Umaro-Jones-FF6

HoxNorf committed:
New Sprites and Filling in Portraits

New Sprites/Palettes (3):
- Blaze-HoxNorf-Sonic
- Toadette (Explorer)-HoxNorf-Mario
- Xion-Kzinssie-KH

New Portraits (6):
- Blaze-HoxNorf-Sonic
- Gau (Suit)-Tsushiy_Gens-FF6
- Gerad-Tsushiy_Gens-FF6
- Toadette (Explorer)-HoxNorf-Mario
- Tohka-HoxNorf-DAL
- Xion-Kzinssie-KH

HoxNorf committed:
Merge branch 'dev' into sprite

asilverthorn committed:
1.3.0 (#64)

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
Merge remote-tracking branch 'upstream/main' into sprite

HoxNorf committed:
New Aesthetic Additions

New Palettes (10):
- Cleon (Player 1)-HoxNorf-PB4
- Cleon (Player 2)-HoxNorf-PB4
- Doom Guy-JamesWhite89-Doom
- Koishi-HoxNorf-Touhou
- Lenna (Dancer)-HoxNorf-FF5
- Lenna (Ranger)-HoxNorf-FF5
- Lenna (Time Mage)-HoxNorf-FF5
- Ness-Badass-Mother
- Satori-HoxNorf-Touhou
- Tohka (Spirit)-HoxNorf-DAL

New Portraits (11):
- Bubble Dragon-Peebs-BB
- Cleon-HoxNorf-PB4
- Doom Guy-JamesWhite89-Doom
- Koishi-HoxNorf-Touhou
- Lenna (Dancer)-JamesWhite89-FF5
- Lenna (Ranger)-JamesWhite89-FF5
- Lenna (Time Mage)-JamesWhite89-FF5
- Ness-Unknown-Mother
- Reimu3-HoxNorf-Touhou
- Satori-HoxNorf-Touhou
- Tohka (Spirit)-HoxNorf-DAL

New Sprites (9):
- Cleon-HoxNorf-PB4
- Doom Guy-JamesWhite89-Doom
- Koishi-HoxNorf-Touhou
- Lenna (Dancer)-HoxNorf-FF5
- Lenna (Ranger)-HoxNorf-FF5
- Lenna (Time Mage)-HoxNorf-FF5
- Ness-Badass-Mother
- Satori-HoxNorf-Touhou
- Tohka (Spirit)-HoxNorf-DAL

asilverthorn committed:
1.3.1 (#99)

- Chest Random Monster bug has been fixed (there was one buggy event encounter that was missed in 1.3.0 and is now removed from the random possibilities)
- New "Auto" status objective results: Dark, Clear, and Imp. Like the others, these get applied at the start of each battle.
- New sprites & portraits

Co-authored-by: HoxNorf <45671870+HoxNorf@users.noreply.github.com>

HoxNorf committed:
Merge branch 'main' into sprite

BriGuy7727 committed:
add Lone Wolf & Tzen checks to NFP

BriGuy7727 committed:
Merge branch 'main' of https://github.com/BriGuy7727/WorldsCollide

BriGuy7727 committed:
remove Tzen Thief, add Mt Zozo to NFP setting

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
New Aesthetic Additions

* New Portraits (6)
- Daryl-IvaarForkbeard-FF6
- Eirin-HoxNorf-Touhou
- Riku-HoxNorf-KH
- Suika-HoxNorf-Touhou
- Umaro (Morph)-CDude-FF6
- Yamame-HoxNorf-Touhou

* New Sprites/Palettes (5):
- Daryl-IvaarForkbeard_Odbarc-FF6
- Eirin-HoxNorf-Touhou]
- Riku-HoxNorf-KH
- Suika-HoxNorf-Touhou
- Yamame-HoxNorf-Touhou

HoxNorf committed:
Merge branch 'dev' into sprite

HoxNorf committed:
Last Few Aesthetic Additions

New Sprites/Palettes (8):
- Samus-Badass-Metroid
- Ridley-JamesWhite89-Metroid
- Marth-HoxNorf-FE
- Bartz (Knight)-HoxNorf-FF5
- Lenna (Knight)-HoxNorf-FF5
- Lenna (Thief)-HoxNorf-FF5
- Lenna (Summoner)-HoxNorf-FF5
- Lenna (Beastmaster)-HoxNorf-FF5

New Portraits (7):
- Samus-Unknown-Metroid
- Ridley-JamesWhite89-Metroid
- Marth-HoxNorf-FE
- Lenna (Knight)-JamesWhite89-FF5
- Lenna (Thief)-JamesWhite89-FF5
- Lenna (Summoner)-JamesWhite89-FF5
- Lenna (Beastmaster)-JamesWhite89-FF5

asilverthorn committed:
1.4.0 (#116)

Feature: Added flags for tiered equipable equipment and tiered equipable relics (doubledown)
Feature: Added flag to customize list of potential item rewards from checks (doubledown)
Feature: Split esper spells flags into separate flags for spells and learn rate (doubledown)
Feature: Throwable and Restoratives objectives (doubledown)
Feature: IronMog Mode flag to prevent saving in seeds (asilverthorn)
Feature: Added High Tier Weapon, High Tier Armor, High Tier Shld, High Tier Helm, and High Tier Relic as objective results (asilverthorn)
QoL: Eliminated in-game RNG with fixed encounter - every runner will encounter the same enemies at the same fixed encounters (asilverthorn)
QoL: Eliminated random fixed encounters on Lete River - now, Left/Right will always be 4 encounters, Straight will be 5. Going Up will add 2 encounters. (asilverthorn). Ref: Eliminating in-game RNG in fixed encounters #114
QoL: Reduced Poison pixelation (franklin)
QoL: Add Mt. Zozo and Lone Wolf to No Free Characters/Espers (TheShwantz27)
Bugfix: Fixed Relm's theme playing at the Owzer's Mansion if it's a different character (asilverthorn)
Other: Removed the L. from "Remove L. Spells" in the flags menu to avoid confusion (asilverthorn)
Other: Deprecated Random RNG (-rr) flag - it's always on (doubledown)
Other: Deprecated Randomize Clock (-rc) flag - it's always on (doubledown)
Other: Made Lone Wolf character reward play the character's theme and Updated Terra's theme to "Terra's Theme" from "Awakening" (asilverthorn)
Other: Locke's Theme will continue to play through Narshe Moogle Defense non-boss battles (asilverthorn)
Other: New Sprites and Portraits (HoxNorf)

BriGuy7727 committed:
Merge branch 'main' of https://github.com/BriGuy7727/WorldsCollide

BriGuy7727 committed:
fix bug with Phoenix Cave & KT skip for mid path

BriGuy7727 committed:
fix bug with Phoenix Cave & KT skip for mid path

asilverthorn committed:
1.4.1 (#118)

* fix bug with Phoenix Cave & KT skip for mid path
---------

Co-authored-by: Brian - TheShwantz27 <briguy7727@yahoo.com>

BriGuy7727 committed:
Merge pull request #1 from ff6wc/main

1.4.1 (#118)

asilverthorn committed:
1.4.2 - Check objective rework (#120)

* moving KT non-checks to quests; adding AH checks to check objective list; eliminating lines for 'All 1 of' objectives

rblakedavis committed:
Update arguments.py

allow for -o to handle directories. (gives the default filename at the specified directory if no filename is given)

HoxNorf committed:
Merge branch 'main' into sprite

HoxNorf committed:
New Aesthetic Update

New Sprites/Palettes (9):
- Flowey-pomariin-Undertale
- Kirby-HoxNorf-Kirby
- Miko-HoxNorf-Touhou
- Mokou-HoxNorf-Touhou
- Purple Heart-HoxNorf-Touhou
- Reimu (PC98)-HoxNorf-Touhou
- Shulk-HoxNorf-Xenoblade
- Toadette (Sailor)-HoxNorf-Mario
- Youmu-HoxNorf-Touhou

Updated Sprites (4):
- Hotaru-HoxNorf-MOTW
- Peach-Halkel-SMRPG
- Toadette (Explorer)-HoxNorf-Mario
- Toadette-HoxNoed-Mario

New Portraits (9):
- Flowey-pomariin-Undertale
- Kirby-Unknown-Kirby
- Miko-HoxNorf-Touhou
- Mokou-HoxNorf-Touhou
- Purple Heart-HoxNorf-Touhou
- Reimu (PC98)-HoxNorf-Touhou
- Shulk-HoxNorf-Xenoblade
- Toadette (Sailor)-HoxNorf-Mario
- Youmu-HoxNorf-Touhou

Removed Portraits (1):
- Wolf-JamesWhite89-Startfox (unneeded duplicate due to a typo of Wolf-JamesWhite89-Starfox)

HoxNorf committed:
New Additions

* New Sprites/Palettes (2):
- Meiling-HoxNorf-Touhou
- Olimar-HoxNorf-Pikmin

* New Portraits (2):
- Meiling-HoxNorf-Touhou
- Olimar-HoxNorf-Pikmin

BriGuy7727 committed:
Merge branch 'main' of https://github.com/BriGuy7727/WorldsCollide

BriGuy7727 committed:
update Coliseum display for shuffle+random

BriGuy7727 committed:
update Coliseum display for shuffle+random, include %s

BriGuy7727 committed:
update display for Coliseum opponents & rewards

HoxNorf committed:
More Additions

* New Sprites/Palettes (4):
- Alys-HoxNorf-PS4
- Chaz-HoxNorf-PS4
- Tao-HoxNorf-Shining
- Tycoon-HoxNorf-FF5

* New Portraits (4):
- Alys-HoxNorf-PS4
- Chaz-HoxNorf-PS4
- Tao-HoxNorf-Shining
- Tycoon-metalliguy_Gens-FF5

HoxNorf committed:
More Additions for Graphics

* New Sprites/Palettes (5):
- Demi-HoxNorf-PS4
- Falco-Badass-Starfox
- Fox-CtrlxZ-Starfox
- Lutz-HoxNorf-PS1
- Yukari-HoxNorf-Touhou

* New Portraits (5):
- Demi-HoxNorf-PS4
- Falco-JamesWhite89-Starfox
- Fox-JamesWhite89-Starfox
- Lutz-HoxNorf-PS1
- Yukari-HoxNorf-Touhou

asilverthorn committed:
Fixing bug where multiple items can be received from PT reward

asilverthorn committed:
Merge remote-tracking branch 'BriGuy7727/main' into 1.4.3

asilverthorn committed:
Removing Mt. Zozo from No Free Characters/Espers (reverting 1.4.0) due to it breaking 100% seeds

asilverthorn committed:
Adding -scan to output logs when set

asilverthorn committed:
Merge remote-tracking branch 'rblakedavis/main' into 1.4.3

asilverthorn committed:
Merge remote-tracking branch 'HoxNorf/sprite' into 1.4.3

asilverthorn committed:
Removing debug output and increasing version

dbldown11 committed:
Removed chance of Dried Meat being placed in Figaro Castle WOR shops

dbldown11 committed:
Removed reference to depreciated code from Phantom Train check and expanded -sdm input range

asilverthorn committed:
Merge remote-tracking branch 'dbldown11/main' into 1.4.3

asilverthorn committed:
Supporting newer versions of python

beauxq commented on data/espers.py:
```diff
@@ -345,7 +345,7 @@ def get_random_esper(self):
             return None
 
-        rand_esper = random.sample(self.available_espers, 1)[0]
+        rand_esper = random.sample(list(self.available_espers), 1)[0]
```
```suggestion
        rand_esper = random.sample(sorted(self.available_espers), 1)[0]
```
The order of iterating through a set is not deterministic.
If we want to make sure it gets the same result every time with a given seed, we have to make sure the order is the same every time.

asilverthorn replied:
thanks, changed

asilverthorn committed:
switching from list to sorted to ensure deterministic behavior. Thanks beauxq

HoxNorf committed:
New Aesthetic Additions (1.4.3)

New Sprites/Palettes (6):
- Bartz (Dancer)-HoxNorf-FF5
- Bow-HoxNorf-Mario
- Ike (Ranger)-HoxNorf-FE
- Marisa2-HoxNorf-Touhou
- Megumin-HoxNorf-Konosuba
- Tewi-HoxNorf-Touhou

Updated Sprites/Palettes (1):
- Iffy-HoxNorf-Neptunia

New Portraits (4):
- Bow-HoxNorf-Mario
- Ike (Ranger)-HoxNorf-FE
- Megumin-HoxNorf-Konosuba
- Tewi-HoxNorf-Touhou

BriGuy7727 committed:
prevent random boss + Unlock Kefka requirements from softlocks

BriGuy7727 committed:
prevent random boss + Unlock Kefka requirements from softlocks

BriGuy7727 committed:
prevent random boss + Unlock Kefka requirements from softlocks

BriGuy7727 committed:
prevent random boss + Unlock Kefka requirements from softlocks

asilverthorn committed:
Merge pull request #18 from BriGuy7727/1.4.3

Trello Card 84: prevent random boss + Unlock Kefka requirements from softlocks

asilverthorn committed:
Merge pull request #17 from HoxNorf/sprite

New Aesthetic Additions (1.4.3)

ff6wc committed:
Merge branch 'dev' into 1.4.3
